eda_label_distribution.py

```diff
@@ -0,0 +1,132 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
```

Comment:
What do you think of adding a short header to the notebook? 
Something like this (that's what you already commented in one of my notebooks :)
```
# %% [markdown]
# #Eda issue 39 distribution of lables #39
# Issue link: https://github.com/dominikmn/one-million-posts/issues/39
```